devicesetup/build.gradle

```diff
@@ -1,7 +1,9 @@
 apply plugin: 'com.android.library'
+apply plugin: 'maven'
+group = 'com.github.Sushinn'
 
 // This is the library version used when deploying the artifact
-version = '0.3.6'
+version = '0.3.7'
```

Comment:
You can keep the version bump here, but I've got a few more changes I want to make before I do another release (plus I want to test this a bit more myself, esp in combination with my additional changes).